### ISSUE TITLE
Image missing alt text

### ISSUE THREAD

**Issue by discospiff** [mikeseitz1/IS7012#2]
Use the alt attribute on the img tag to describe the image to a user who can't see it.  Describe, in a concise way, what the image looks like.

https://github.com/mikeseitz1/IS7012/blob/9370bc7339cd229607de78f29e6620f5f7856fa2/Week1/seitzme-about-me.html#L18
